systemtests/network_test.go

```diff
@@ -331,8 +331,10 @@ func (s *systemtestSuite) testNetworkAddDeleteTenant(c *C, encap string) {
 		}
 
 		for tenant, networks := range tenantNames {
+			logrus.Infof("\nThis is tenantNames:%s\n", tenantNames)
```

Comment:
If this log is needed can you please remove this ? I think we have a log in tenant create,